src/OpenTelemetry.Resources.Azure/ResourceAttributeConstants.cs

```diff
@@ -27,6 +27,9 @@ internal sealed class ResourceAttributeConstants
     internal const string AzureContainerAppsReplicaNameEnvVar = "CONTAINER_APP_REPLICA_NAME";
     internal const string AzureContainerAppsRevisionEnvVar = "CONTAINER_APP_REVISION";
 
+    internal const string AzureContainerAppJobNameEnvVar = "CONTAINER_APP_JOB_NAME";
+    internal const string AzureContainerAppJobExecutionNameEnvVar = "CONTAINER_APP_JOB_EXECUTION_NAME";
```

Comment:
nit: needs a comment labeling these variables, same as the other groups.
```suggestion
    // Azure Container Apps Jobs environment variables
    internal const string AzureContainerAppJobNameEnvVar = "CONTAINER_APP_JOB_NAME";
    internal const string AzureContainerAppJobExecutionNameEnvVar = "CONTAINER_APP_JOB_EXECUTION_NAME";
```